Src/Core/MainMenu/MainMenu.hpp

```diff
@@ -9,7 +9,7 @@
 
 #include "../../Games/Common/Button.hpp"
 #include "IGame.hpp"
-#include "IGraph.hpp"
+#include <vector>
 
 class MainMenu : public IGame {
```

Comment:
don't forget to add 
/// \file MainMenu.hpp
and a 
/// \brief short desc abt MainMenu class